6_visualize.yml

```diff
@@ -32,10 +32,8 @@ targets:
   6_visualize:
     depends:
       - 6_storm_gif_tasks.yml
-      - 6_visualize/log/6_storm_gif_tasks.ind
-      - 6_intro_gif_tasks.yml
-      - 6_visualize/log/6_intro_gif_tasks.ind
-      - 6_visualize/out/animation_a.gif
+      - storm_gif_tasks_executed
```

Comment:
since `6_visualize/out/animation_a.mp4` depends on `storm_gif_tasks_executed`, you shouldn't need to include it here too (right?)

Reply:
Oh oops, you're right. Added it at the top first because I was trying to get it to run when I just did `scmake('6_visualize/out/animation_a.mp4', '6_visualize.yml', force=TRUE)` and remembered the `depends` trick for individual targets after. Removing!